engine/src/main/java/org/terasology/rendering/dag/nodes/OpaqueBlocksNode.java

```diff
@@ -48,41 +48,31 @@
  * In a typical world this is the majority of the world's landscape.
  */
 public class OpaqueBlocksNode extends AbstractNode implements WireframeCapable {
-    private static final ResourceUrn CHUNK_SHADER = new ResourceUrn("engine:prog.chunk");
+    private static final ResourceUrn CHUNK_MATERIAL = new ResourceUrn("engine:prog.chunk");
 
-    @In
     private WorldRenderer worldRenderer;
-
-    @In
-    private Config config;
-
-    @In
     private RenderQueuesHelper renderQueues;
 
-    @In
-    private DisplayResolutionDependentFBOs displayResolutionDependentFBOs;
-
     private Camera playerCamera;
     private Material chunkShader;
     private SetWireframe wireframeStateChange;
     private RenderingDebugConfig renderingDebugConfig;
 
-    /**
-     * Initialises this node. -Must- be called once after instantiation.
-     */
-    @Override
-    public void initialise() {
+    public OpaqueBlocksNode(Context context) {
+        renderQueues = context.get(RenderQueuesHelper.class);
+        worldRenderer = context.get(WorldRenderer.class);
+
         wireframeStateChange = new SetWireframe(true);
-        renderingDebugConfig = config.getRendering().getDebug();
+        renderingDebugConfig = context.get(Config.class).getRendering().getDebug();
```

Comment:
I didn't highlight this issue in previous nodes as it isn't overly important but perhaps we might want to decide if to consistently initialize the two rendering configs either by first obtaining the Config object from the context or, as you do here, by having everything in one line. 

I guess on one hand I feel that having a local variable helps with the readability given that `getRendering()` and `getDebug()` are a bit ambiguous as method names go. On the other I like the compactness of the one-line in this case.

I'd say I'd prefer readability over compactness though. What do you think?